samples/sampler/stories/default/Spinner.js

```diff
@@ -1,15 +1,32 @@
 import {mergeComponentMetadata} from '@enact/storybook-utils';
 import {boolean, select} from '@enact/storybook-utils/addons/knobs';
+import kind from '@enact/core/kind';
 import UiSpinner, {SpinnerBase as UiSpinnerBase} from '@enact/ui/Spinner';
 import ri from '@enact/ui/resolution';
 import React from 'react';
+import PropTypes from 'prop-types';
 import {storiesOf} from '@storybook/react';
 
+import Skinnable from '@enact/agate/Skinnable';
 import Spinner, {SpinnerBase} from '@enact/agate/Spinner';
 
 Spinner.displayName = 'Spinner';
 const Config = mergeComponentMetadata('Spinner', UiSpinnerBase, UiSpinner, SpinnerBase, Spinner);
 
+const SkinSpinnerBase = kind({
```

Comment:
@adrian-cocoara-lgp
It looks better to rename this to `SkinnedSpinnerBase` just like a SkinnedIcon in Icon sampler and a SkinnedButton in a button sampler. :)